packages/mantine/src/components/table/TableActions.tsx

```diff
@@ -27,7 +36,8 @@ interface TableActionsProps<T> {
     children: ((datum: T) => ReactNode) | ((data: T[]) => ReactNode);
 }
 
-export const TableActions = <T,>({children}: TableActionsProps<T>): ReactElement => {
+export const TableActions = <T,>({children, classNames, styles, unstyled}: TableActionsProps<T>): ReactElement => {
```

Comment:
We should probably also spread `...others` on the root element so that props like `p`, `m`, or `sx` can work.

Reply:
Good call 👍